keps/prod-readiness/sig-windows/2258.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 2258
 alpha:
   approver: "@johnbelamaric"
+beta:
+  approver: "@deads2k"
```

Comment:
Is there a specific reason to switch to David? We usually try to keep it the same just since the previous reviewer will have context already.

Reply:
There is no specific reason other than that David [pinged](https://kubernetes.slack.com/archives/CPNHUMN74/p1707166593892599) me on the k8s slack asking about the KEP. So I assumed PRR review transferred to him. Apologies for making that assumption. I have made you the approver.